src/styles.scss

```diff
@@ -1674,6 +1693,11 @@ $green-font-color: #19B028;
   color: $important-text-color;
   font-size: $normal-text-size;
 }
+.form-control {
+  color: var(--text) !important;
+  background: var(--background) !important;
+  border-color: var(--secondary) !important;
```

Comment:
`form-control` shouldn't use the same background color as the surface background. The style used elsewhere is

```
background-color: #fafafa;
border: none;
```